### ISSUE TITLE
Interpolated address not found in Resistencia, Argentina.

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#5203]
**[Submitted to the original trac issue database at 1.59am, Friday, 25th July 2014]**

The query  
http://nominatim.openstreetmap.org/search.php?q=santiago+de+liniers+50%2C+resistencia%2C+argentina
is not returning the expected result.

The expected result is some point in the middle of the interpolation way http://www.openstreetmap.org/way/114708392

In the same street, for address numbers > 98 the query returns ok, and also works ok for odd address numbers in the 1-99 range.

I have searched for problems in OSM data but I hadn't found it. I don't know if the problem is in the OSM data or in nominatim.
I have been fixing some address data in the area, some problems were corrected and some searchs works well, but the problem with the reported query still remains.

**Comment by tomhughes** [comment]
**Author: lonvia**
**[Added to the original trac issue at 8.36pm, Friday, 25th July 2014]**

That's two issues with Nominatim here.

First, Nominatim doesn't like address interpolations that start with 0. There have been some discussions with South American mappers a while ago and we came to the conclusion that this is actually the correct tagging in some parts (I think it is even written like this on the street signs). So, this needs to be fixed.

The second problem is that Nominatim doesn't update interpolation if only the nodes are changed. So, it didn't catch it when you changed the start number to 2. I've fixed the example for now manually but something needs to be done about that as well.
